### PR TITLE
[EXP-62-262] split s3 script into endorsed/experimental

### PR DESCRIPTION
**Pull request by mariuspod**

### CODE REVIEW

**Review by jstashh — APPROVED** [review]
<3

**Comment by mariuspod** [comment]
@jstashh pls lmk what you think about this separation. I tried it locally and was able to run the mainnet eth s3 script for only the endorsed vaults in 6-8 minutes.

File: scripts/s3.py [hunk 1]
```diff
@@ -170,30 +175,42 @@ def main():
...
-    file_path = os.path.join(out, vaults_api_path)
-    os.makedirs(file_path, exist_ok=True)
+    file_name, s3_path = _get_export_paths(suffix)
+    _export(to_export, file_name, s3_path)
 
-    endorsed = [vault for vault in data if vault["endorsed"]]
-    experimental = [vault for vault in data if not vault["endorsed"]]
+    # Sent a metric so we can track and alert on if this was successfully generated
+    utc_now = datetime.utcnow()
+    send_metric(f"{METRIC_NAME}.success", 1, utc_now, tags=metric_tags)
```
Comment (jstashh) [inline scripts/s3.py:195]: Should we add what kind of export_mode we're using in `metric_tags`? Likewise for the telegram message would be good to include whether endorsed or experimental is being released

**Review by jstashh — COMMENTED** [review]
lgtm, just added the comment about monitoring messages

**Comment by mariuspod** [comment]
@jstashh ty for your review, updated!

**Pull request merged**